daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -2127,25 +2132,10 @@ private[lf] object SBuiltin {
     override protected def executeUpdate(
         args: util.ArrayList[SValue],
         machine: UpdateMachine,
-    ): Control[Nothing] = {
+    ): Control.Value = {
```

Comment:
Here the whole contract is pass as argument to the builtin. Then we do not need the disclosure table to be prefilled.